### PR TITLE
Update to Facebook Airlift

### PR DESCRIPTION
**Pull request by wenleix**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by wenleix** [comment]
Travis turns green previously 😃 . Just rebased and see if it still works

**Comment by arhimondr** [comment]
It actually looks like a bug in the dependency analyzer plugin. 

We can investigate it further. As for this PR i think we should have a workaround. 

However instead of declaring this dependencies i think it is better to have an exclusion. Here is the doc: https://maven.apache.org/plugins/maven-dependency-plugin/analyze-mojo.html#ignoredUsedUndeclaredDependencies

Also please add a comment explaining what is going on next to the exclusion.

File: presto-orc/pom.xml [hunk 1]
```diff
@@ -186,6 +186,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
+            <version>3.4</version>
```
[the remark this answers is not in the document]
Comment (wenleix) [reply presto-orc/pom.xml]: @arhimondr : See https://github.com/prestosql/presto/pull/25/commits/ad3b4ac94093e6f942bbc755da7c7703158859da#diff-066d62f9dfcedd1ce566d091d87c9f1eR235 :)

File: presto-docs/src/main/resources/logo/web/fb/black/Presto_FB_Lockups_BLACK_BG-25.svg [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Generator: Adobe Illustrator 17.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 width="384.002px" height="192.001px" viewBox="0 0 384.002 192.001" enable-background="new 0 0 384.002 192.001"
-	 xml:space="preserve">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
```
[the remark this answers is not in the document]
Comment (wenleix) [reply presto-docs/src/main/resources/logo/web/fb/black/Presto_FB_Lockups_BLACK_BG-25.svg]: @arhimondr : Going to revert it :)

**Comment by wenleix** [comment]
Here are the dependencies removed per discussion (instead, using `ignoredDependency` for now)
```
        <dependency>
            <groupId>org.glassfish.jersey.core</groupId>
            <artifactId>jersey-common</artifactId>
            <version>2.26</version>
            <exclusions>
                <exclusion>
                    <groupId>org.glassfish.hk2.external</groupId>
                    <artifactId>javax.inject</artifactId>
                </exclusion>
            </exclusions>
            <scope>test</scope>
        </dependency>

        <dependency>
            <groupId>org.eclipse.jetty</groupId>
            <artifactId>jetty-server</artifactId>
            <version>9.4.14.v20181114</version>
            <scope>test</scope>
        </dependency>
```

**Pull request merged**